packages/cozy-harvest-lib/src/services/updateAccountsPassword.js

```diff
@@ -0,0 +1,77 @@
+import SymmetricCryptoKey from 'cozy-keys-lib/transpiled/SymmetricCryptoKey'
+import get from 'lodash/get'
+import set from 'lodash/set'
+import unset from 'lodash/unset'
+
+const decryptString = (encryptedString, vaultClient, orgKey) => {
+  const [encTypeAndIv, data, mac] = encryptedString.split('|')
+  const [encTypeString, iv] = encTypeAndIv.split('.')
+  const encType = parseInt(encTypeString, 10)
+
+  return vaultClient.cryptoService.aesDecryptToUtf8(
+    encType,
+    data,
+    iv,
+    mac,
+    orgKey
+  )
+}
+
+const updateAccountsPassword = async (
+  cozyClient,
+  vaultClient,
+  bitwardenCipherDocument
+) => {
+  const cozyKeys = await cozyClient
+    .getStackClient()
+    .fetchJSON('GET', '/bitwarden/organizations/cozy')
+
+  const encryptedPassword = get(bitwardenCipherDocument, 'login.password')
+  const encryptedUsername = get(bitwardenCipherDocument, 'login.username')
+  const bitwardenCipherId = get(bitwardenCipherDocument, '_id')
+
+  const orgKeyEncType = 2
```

Comment:
```suggestion
  const orgKeyEncType = EncryptionType.AesCbc256_HmacSha256_B64
```

avec l'import qui va bien